### PR TITLE
feat(font): add sn-pro (#65)

### PR DESCRIPTION
**Pull request by acnebs**

### CODE REVIEW

File: fonts/other/sn-pro/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "weights": [200, 300, 400, 500, 600, 700, 800, 900],
+  "styles": ["normal", "italic"],
+  "defSubset": "latin",
+  "variable": false,
+  "lastModified": "2024-04-24",
+  "version": "v1.1.0",
+  "category": "sans-serif",
+  "license": {
+    "type": "OFL-1.1",
+    "url": "https://github.com/supernotes/sn-pro",
```
Comment (ayuhito) [inline fonts/other/sn-pro/metadata.json:14]:
```suggestion
    "url": "https://github.com/supernotes/sn-pro/blob/main/OFL.txt",
```

This should be a link to the license directly.

**Review by ayuhito — CHANGES_REQUESTED** [review]
Everything looks good to me! Just a small nit before I can merge this 🎉

**Comment by acnebs** [comment]
Done!

**Review by ayuhito — APPROVED** [review]
LGTM! Thank you 🎉

**Comment by ayuhito** [comment]
It's up now. It takes around 2-8 hours for our caches to expire on the API service and so while our search indexes update instantly, we haven't got anything setup to invalidate the API caches.

**Pull request merged**